packages/ws-server/src/data-source.ts

```diff
@@ -178,5 +153,5 @@ export function createWSServerDataSource() {
     return () => handlers.delete(handler)
   }
 
-  return { update, onActionEvent, subscribe, get, updateRoot, handleWSConnection }
```

Comment:
Should we bring `onEvent` back, for side-effect console logging and debugging purposes? Would get called on each message, but would have no impact on the actual return to the client

Reply:
Decided - YES

Reply:
yes I think a lower-level onEvent would be great! we can do it in a follow-up PR